app/controllers/candidate_interface/work_history/length_controller.rb

```diff
@@ -0,0 +1,27 @@
+module CandidateInterface
+  class WorkHistory::LengthController < CandidateInterfaceController
+    def length
+      @work_details_form = WorkHistoryForm.new
+    end
+
+    def submit_length
+      @work_details_form = WorkHistoryForm.new(work_history_params)
+
+      if @work_details_form.valid?
+        redirect_to candidate_interface_work_history_new_path
+      else
+        render :length
+      end
+    end
+
+  private
+
+    def work_history_params
+      return nil unless params.has_key?(:candidate_interface_work_history_form)
```

Comment:
do we need this? it sounds a bit of a `hack`

Reply:
Yes, it's because of the form not having any fields that have any defaults. The alternative is to add a hidden field, but I'm not sure if that's preferable.